### DISCUSSION TITLE
any available reporter?

### DISCUSSION THREAD

**Discussion by yuen07** [zricethezav/gitleaks#568] [Q&A] [answered]
Hi, the results of the tool comes in a json format, but is there a way to present this in more friendly manner? or a tool that is a viewer on a 3rd party?

**Comment by w0rmr1d3r** [comment, marked as answer]
Hello @yuen07 !
If you take a look at this section -> https://github.com/zricethezav/gitleaks#usage-and-options
It can output content in CSV as well and SARIF.
Which formats are you missing?

**Reply by yuen07** [reply]
maybe im using it wrong, when i put this gitleaks --repo-url=repourl -v --access-token=accesstoken--report=results.csv --format=CSV it comes back blank :/

**Reply by w0rmr1d3r** [reply]
Don't you worry! There's an open issue about this -> https://github.com/zricethezav/gitleaks/issues/524
As said in the issue, you might wanna try the same command but the format in lower case, like this -> `--format=csv`
They are working on a fix to make it easier and clearer!

**Reply by w0rmr1d3r** [reply]
The issue has been solved with this https://github.com/zricethezav/gitleaks/issues/524
